packages/genesys-spark-components/src/components/beta/gux-segmented-control/gux-segmented-control-item/gux-segmented-control-item.tsx

```diff
@@ -113,7 +114,7 @@ export class GuxSegmentedControlItem {
             'gux-selected': this.selected
           }}
           type="button"
-          disabled={this.disabled}
+          disabled={this.disabled || this.hasDisabledParent()}
```

Comment:
I was just wondering if you just `disable` a segmented-control-item it will just disable the button and not the container so the border color looks like its not disabled. But from the designs it looks like they just disable the button element.

Reply:
got the ok for the change from UX

Reply:
I should have caught this earlier but when `disabled` should the tooltip appear it currently does. Rest of changes look good to me 👍

Reply:
I think it appearing is correct too.